tests/main.cpp

```diff
@@ -8,20 +9,28 @@
 #include <stack>
 
 #include <GLFW/glfw3.h>
+#include "../../tangram-es/core/include/glm/glm/glm.hpp"
```

Comment:
Yeah. This would not have gotten into master. Talked with Varun about it - for this kind of stuff I miss our Native flow :(